descartes_utilities/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(descartes_utilities)
+
+find_package(catkin REQUIRED COMPONENTS
+  descartes_core
+  descartes_trajectory
+  trajectory_msgs
+)
+
+catkin_package(
+ INCLUDE_DIRS include
+ LIBRARIES descartes_utilities
+ CATKIN_DEPENDS descartes_core descartes_trajectory trajectory_msgs
+)
+
+include_directories(include)
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_library(${PROJECT_NAME}
+  src/ros_conversions.cpp
+)
+
+add_dependencies(${PROJECT_NAME} 
+  ${${PROJECT_NAME}_EXPORTED_TARGETS} 
+  ${catkin_EXPORTED_TARGETS}
+)
+
+install(TARGETS ${PROJECT_NAME}
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.h"
+  PATTERN ".svn" EXCLUDE
+)
```

Comment:
I know this is in the boilerplate `CMakeLists.txt` that is generated by `catkin_create_pkg`, but the `EXCLUDE` pattern isn't really needed (we don't use subversion in this repository).